README.md

```diff
@@ -103,6 +119,12 @@ We currently bundle the following projects in this image:
 * [amazon-cloudwatch-logs-for-fluent-bit](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit)
 * [amazon-kinesis-streams-for-fluent-bit](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)
 
+### Using the AWS Plugins outside of a container
+
+You can use the AWS Fluent Bit plugins with [td-agent-bit](https://docs.fluentbit.io/manual/v/1.3/installation/td-agent-bit).
+
+We provide a [tutorial](examples/fluent-bit/systems-manager-ec2/) on using SSM to configure instances with td-agent-bit and the plugins.
```

Comment:
I started evaluating this plugin outside of a container just last week; this tutorial was helpful.